app/views/subscription/index.html.slim

```diff
@@ -0,0 +1,6 @@
+.row
+    h1 Subscribe to our newsletter
```

Comment:
let's move this form to the home page - just add input before subscribe to newsletter button